.github/workflows/build_deploy.yml

```diff
@@ -43,11 +51,14 @@ jobs:
         env:
           # configure cibuildwheel to build native archs ('auto'), and some
           # emulated ones
-          CIBW_ARCHS_LINUX: auto aarch64
-          CIBW_ARCHS_MACOS: x86_64 universal2
+          CIBW_ARCHS: ${{ matrix.archs }}
           # PyPY is not supported
           # Windows 2.7 wheels no longer can be built https://github.com/DataDog/dd-trace-py/pull/2344
           CIBW_SKIP: pp* cp27-win*
+          # Run a smoke test on every supported platform
+          CIBW_TEST_COMMAND: python {project}/tests/smoke_test.py
```

Comment:
cool!

Reply:
oh we should totally `DD_TESTING_RAISE` here! (And maybe do a tracer.trace()/profiler.start() as well)